dm/common/common.go

```diff
@@ -80,7 +80,7 @@ var (
 	// k/v: Encode(task-name, downstream-schema-name, downstream-table-name) -> table schema.
 	ShardDDLOptimismInitSchemaKeyAdapter KeyAdapter = keyHexEncoderDecoder("/dm-master/shardddl-optimism/init-schema/")
 	// ShardDDLOptimismDroppedColumnsKeyAdapter is used to store the columns that are not fully dropped
-	// k/v: Encode(task-name, downstream-schema-name, downstream-table-name, column-name, source-id, upstream-schema-name, upstream-table-name) -> empty
```

Comment:
Maybe we should write what's this `integer`'s usage?

Reply:
Besides, if you want to change this key's format, this part must be revised, too.
https://github.com/pingcap/dm/blob/master/pkg/shardddl/optimism/ops.go#L80

Reply:
> Maybe we should write what's this `integer`'s usage?

ditto https://github.com/pingcap/dm/blob/5ae19df6c531537f61d8b961bbbf526fe09fab80/pkg/shardddl/optimism/lock.go#L31-L40

Reply:
> Besides, if you want to change this key's format, this part must be revised, too.
> https://github.com/pingcap/dm/blob/master/pkg/shardddl/optimism/ops.go#L80

address in bf05a9d